packages/world/src/modules/keyswithvalue/KeysWithValueHook.sol

```diff
@@ -29,45 +29,78 @@ contract KeysWithValueHook is IStoreHook {
     return IBaseWorld(StoreSwitch.getStoreAddress());
   }
 
+  function handleSet(bytes32 tableId, bytes32 valueHash, bytes32[] memory key) internal {
```

Comment:
nit: should we call this `_addKeyToList`, move to next to `_removeKeyFromList` and align the function signatures?